terraform/locals.tf

```
@@ -1,4 +1,7 @@
 locals {
-  submitter     = merge(var.submitter, { queue_storage = var.queue_storage }, { tag = var.core_tag })
-  compute_plane = merge(var.compute_plane, { queue_storage = var.queue_storage }, { tag = var.core_tag })
+  replicas        = toset([for s in range(var.num_replicas) : tostring(s)])
+  queue_env_vars  = var.queue_storage.broker.name == "activemq" ? module.queue_activemq[0].queue_env_vars : module.queue_rabbitmq[0].queue_env_vars
```

I would slightly prefer to have this:
```suggestion
  queue = one(concat(module.queue_activemq, module.queue_rabbitmq))
  queue_env_vars = queue.queue_env_vars
```

This would avoid the need for the condition, and would scale better if we have more queues at our disposal.